### DISCUSSION TITLE
Daily report based on catchbook

### DISCUSSION THREAD

**Discussion by Errandir-of-Gondolin** [janstehno/cotw-companion#91] [Ideas]
I often find myself wanting to see a quick overview of what exactly I've shot over a (real-time) day of hunting. In the catchbook you can sort by date, but only a limited amount of catches are visible in the list at any given time (you have to scroll), which means that it's hard to get a real view or an idea about your totals. So some kind of 'report' that shows you a dashboard with useful info on your catches of the day/week/month could be a fun addition. Info such as amount of kills (total and per species), split between male/female, average trophy score per species, etc.

**Comment by janstehno** [comment]
Great idea. Won't be the highest priority, but I will put it to planned features.
